src/ui/src/App.tsx

```diff
@@ -1,7 +1,14 @@
-import Home from './pages/Home/Home';
+import { Routes, Route } from 'react-router-dom';
+import routes from './routes';
 
 export function App() {
-  return <Home />;
+  return (
+    <Routes>
+      {routes.map(({ key, path, component: Component }) => (
+        <Route key={key} path={path} element={<Component />} />
+      ))}
+    </Routes>
```

Comment:
Let's represent a separate component for it, because when we add contexts our `App.tsx` will get messy